README.md

```diff
@@ -32,7 +32,7 @@ Structure of the repository
 Quick start
 -----------
 
-The current version of the plugins requires the version 3.6.1 of the clang compiler, re-compiled from source.
+The current version of the plugins requires the version 3.6.1 of the clang compiler, re-compiled from the source.
```

Comment:
"Compile from source" is the usual expression afaik, e.g. https://golang.org/doc/install/source